rgw/v2/lib/resource_op.py

```diff
@@ -166,6 +166,7 @@ def read(self):
         self.max_objects_per_shard = self.doc['config'].get('max_objects_per_shard')
         self.max_objects = None
         self.user_count = self.doc['config'].get('user_count')
+        self.user_remove = self.doc['config'].get('user_remove')
```

Comment:
I would suggest to have default value as `False` because this will return `None` and else the below code would fail if the `user_remove` key is not added in the other yamls where this test is being called 
```
 if config.user_remove is True:
        reusable.remove_user(each_user)
```

Reply:
Thanks for the review Rakesh. Have updated the changes.